### PR TITLE
Chaos Level

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Adds the chaos level vote into the game. It can be called after an hour and a half from roundstart and after that every 30 minutes. Each vote increases storyteller point generation (first vote OG value * 2, second vote OG value * 3 and so on), bluespace entropy generation and lowers cooldowns on bluespace entropy events.
Player vote need 75% positive votes to pass as it increases point generation for the rest of the round.
Bandaid fix until storyteller rework (never)

## Why It's Good For The Game

Speeds up rounds after the 90 minute mark, and only if the players want so. Many a time have I found myself at this time of the game after a major threat has been defeated (excels, hive, serbs), with the choice to wait another hour so something interesting would happen or jump now and wait again, but in the next round. Now there is a third choice.

## Changelog
:cl:
add: Added chaos level that can be increased by player vote, that increases storyteller point generation.
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
TERROR LEVEL RAISED

**Comment by TheShown911** [comment]
Added admin buttons in storyteller panel.

File: code/__HELPERS/bluespace_entropy.dm [hunk 1]
```diff
@@ -12,20 +12,20 @@ GLOBAL_VAR_INIT(bluespace_distotion_cooldown, 10 MINUTES)
...
-			GLOB.bluespace_distotion_cooldown = world.time + 5 MINUTES
+			GLOB.bluespace_distotion_cooldown = world.time + (5 MINUTES / GLOB.chaos_level)
 			A.bluespace_entropy -= rand(A.bluespace_hazard_threshold, A.bluespace_hazard_threshold*1.5)
 			bluespace_distorsion(T, minor_distortion)
 	else
 		GLOB.bluespace_entropy += entropy_value
 		var/entropy_cap = rand(GLOB.bluespace_hazard_threshold, GLOB.bluespace_hazard_threshold*2)
 		if(GLOB.bluespace_entropy >= entropy_cap && world.time > GLOB.bluespace_distotion_cooldown)
-			GLOB.bluespace_distotion_cooldown = world.time + 10 MINUTES
+			GLOB.bluespace_distotion_cooldown = world.time + (10 MINUTES * GLOB.chaos_level)
```
Comment (hyperioo) [inline code/__HELPERS/bluespace_entropy.dm:28]:
```suggestion
			GLOB.bluespace_distotion_cooldown = world.time + (10 MINUTES / GLOB.chaos_level)
```
Not a divide? More chaos -> reduced cooldown, like you did for minor distortion

File: code/controllers/subsystems/voting/poll_types.dm [hunk 2]
```diff
@@ -232,6 +232,31 @@
...
 
 
+/datum/poll/chaos_level_increase
+	name = "Increase Chaos Level"
+	question = "Do you want to increase the chaos level?"
+	description = "Higher chaos level makes storyteller events much more likely."
+	time = 120
+	minimum_win_percentage = 0.75 //High % needed for something that alters the whole round
+	cooldown = 30 MINUTES
+	next_vote = 90 MINUTES //Same lenght as bluspace jump
```
Comment (hyperioo) [inline code/controllers/subsystems/voting/poll_types.dm:242]:
```suggestion
	next_vote = 90 MINUTES //Same lenght as bluespace jump
```

**Review by hyperioo — APPROVED** [review]

**Pull request merged**